corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -19,7 +21,8 @@ hqDefine("geospatial/js/geospatial_map", [
 
             var self = {};
             let clickedMarker;
-            mapboxgl.accessToken = initialPageData.get('mapbox_access_token');
+//            mapboxgl.accessToken = initialPageData.get('mapbox_access_token');
+            mapboxgl.accessToken = 'pk.eyJ1Ijoic21pdHRpZWMiLCJhIjoiY2xpOGVzdWIxMXJtczNobndzbzI2ajVwbSJ9.HK2WrNNjEg7khoQ3RrxgYQ';
```

Comment:
Is this intentional?

Reply:
That's a test key...oops. Thanks for raising.